Sources/StreamChat/APIClient/Endpoints/Payloads/CurrentUserPayloads.swift

```diff
@@ -67,6 +71,7 @@ class CurrentUserPayload: UserPayload {
         devices = try container.decodeIfPresent([DevicePayload].self, forKey: .devices) ?? []
         mutedUsers = try container.decodeIfPresent([MutedUserPayload].self, forKey: .mutedUsers) ?? []
         mutedChannels = try container.decodeIfPresent([MutedChannelPayload].self, forKey: .mutedChannels) ?? []
+        blockedUsers = try container.decodeIfPresent([BlockedUserPayload].self, forKey: .blockedUsers) ?? []
```

Comment:
According to the spec, we will also need to include the blocked users in the `Member` models

Reply:
Done 👍